packages/web-components/src/components/ic-menu/test/basic/ic-menu.spec.tsx

```diff
@@ -608,13 +609,123 @@ describe("ic-menu in isolation", () => {
 
     page.rootInstance.isSearchableSelect = true;
 
+    await page.rootInstance.handleSetFirstOption();
+
     page.rootInstance.manSetInputValueKeyboardOpen(keyboardEvent("Backspace"));
 
     await page.waitForChanges();
 
     expect(searchableSelect.setFocus).toHaveBeenCalled();
   });
-  it("tests manSetInputValueKeyboardOpen function when searchbar", async () => {
+
+  it("tests setFocus in not called when backSpace is pressed and no menu option is highlighted", async () => {
```

Comment:
tiny typo, should be 'is not called' 
this won't stop me approving though

Reply:
Happy to make the update